pinot-core/src/test/java/org/apache/pinot/core/data/manager/realtime/Fixtures.java

```diff
@@ -0,0 +1,156 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.core.data.manager.realtime;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Random;
+import org.apache.pinot.core.realtime.impl.fakestream.FakeStreamConsumerFactory;
+import org.apache.pinot.core.realtime.impl.fakestream.FakeStreamMessageDecoder;
+import org.apache.pinot.spi.config.table.TableConfig;
+import org.apache.pinot.spi.data.Schema;
+import org.apache.pinot.spi.data.readers.GenericRow;
+import org.apache.pinot.spi.utils.JsonUtils;
+
+
+public class Fixtures {
+  private Fixtures() {
+  }
+  public static final int MAX_ROWS_IN_SEGMENT = 250000;
+  public static final long MAX_TIME_FOR_SEGMENT_CLOSE_MS = 64368000L;
+  public static final String TOPIC_NAME = "someTopic";
+  public static final String CONSUMER_FACTORY_CLASS = FakeStreamConsumerFactory.class.getName();
+  public static final String MESSAGE_DECODER_CLASS = FakeStreamMessageDecoder.class.getName();
+
+  //@formatter:off
+  public static final String TABLE_CONFIG_JSON =
+      "{"
+          + "  \"metadata\":{},"
```

Comment:
(minor) Let's keep the original indentation for readability

Reply:
Fixed.